app/models/facility.rb

```diff
@@ -1,3 +1,3 @@
 class Facility < ActiveRecord::Base
-    has_many :bills
+    has_many :bill
```

Comment:
I think this should be plural `has_many :bills` is there a reason why you changed to singular?

Reply:
Complete accident